### ISSUE TITLE
patch citybase refund transactions records

### ISSUE THREAD

**Issue by chiaberry** [cityofaustin/atd-data-tech#10920]
- include same transaction id
- customer name
- connection field isnt populating
- is subdescription working?

**Comment by patrickm02L** [comment]
@chiaberry what's the status on this Issue. Diana says she remembers doing this already. Please confirm.

**Comment by chiaberry** [comment]
@patrickm02L I have not finished this yes. This issue is describing the payload that is sent along to a refund being recorded in the transactions table. It's handled by the postback. If refunds are being recorded correctly as is, let me know and I'll close this issue and not update the postback.

**Comment by chiaberry** [comment]
@patrickm02L let me know if this no longer needs to be done, I'm going to pick it back up soon.

**Comment by chiaberry** [comment]
I've updated the postback to store the subdescription field and confirmed the customer name is being stored as well. I updated the code to store the connected record ID, but not sure if its working on the knack side since its a connected record. 

We don't store the transaction ID on the transaction table (object_180), confirmed the same ID is being stored on the messages table for refunds and other payment statuses.
